vimrc

```diff
@@ -222,7 +222,7 @@ let g:go_highlight_trailing_whitespace_error = 0
 
 let test#strategy = "vimux"
 let test#custom_runners = {}
-let test#python#runner = 'nose'
+let test#python#runner = 'pytest'
```

Comment:
We could set this based on the current folders name so that the server sdk still uses nose.

Reply:
I think this works:
```vim
let test_nose = system("nosetests --version")

if v:shell_error
  let test#python#runner = 'pytest'
else
  let test#python#runner = 'nose'
endif
```

Reply:
This doesn't quite work since we run the tests for braintree_python from a docker container. We can expand on this a bit and check if `requirements.txt` contains a `nose` requirement. 

I'm not very good at vimscript, so feel free to modify this but this is currently working for me:

```vim
let test#python#runner = 'pytest'

if filereadable(expand('requirements.txt'))
  let nose_test = system('grep -q "nose" requirements.txt')

  if v:shell_error == 0
    let test#python#runner = 'nose'
  endif
endif
```

This is making an assumption that vim is launched at braintree_python's root dir, which I think is fine.

Reply:
Do you even need to check for the existence of `requirements.txt`? The grep might be enough?